### PR TITLE
[13.0][FIX] base_comment_template: Fix tests: test_company_general_template

### PR DESCRIPTION
**Pull request by victoralmau**
Fix tests: `test_company_general_template`.

Please @CarlosRoca13  and @ernestotejeda can you review it?

@Tecnativa TT29138

### CODE REVIEW

**Review by CarlosRoca13 — APPROVED** [review]
LGTM :+1:

**Review by chienandalu — APPROVED** [review]

File: base_comment_template/tests/test_base_comment_template.py [hunk 1]
```diff
@@ -15,19 +16,20 @@ class TestCommentTemplate(common.SavepointCase):
...
-        cls.user_obj = cls.env["ir.model"].search([("model", "=", "res.users")])
-
+        cls.user_obj = cls.env.ref("base.model_res_users")
         cls.user = cls.env.ref("base.user_demo")
         cls.user2 = cls.env.ref("base.demo_user0")
         cls.partner_id = cls.env.ref("base.res_partner_12")
         cls.partner2_id = cls.env.ref("base.res_partner_10")
-        cls.company = cls.env["res.company"].search([], limit=1)
+        cls.main_company = cls.env.ref("base.main_company")
+        cls.company = cls.env["res.company"].create({"name": "Test company"})
```
Comment (pedrobaeza) [inline base_comment_template/tests/test_base_comment_template.py]: Please do no create new company, as other problems may arise (required fields not filled, etc).

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by ernestotejeda — APPROVED** [review]

File: base_comment_template/tests/test_base_comment_template.py (hunk 1 above)
Comment (victoralmau) [reply base_comment_template/tests/test_base_comment_template.py]: In these use case i need to have created 2 companies, maybe `.copy()` it's a good idea?

Comment (pedrobaeza) [reply base_comment_template/tests/test_base_comment_template.py]: Why 2 companies are needed? Maybe regression tests are going too deep... Anyway, `copy` can prevent such problem, but I prefer to not require 2 companies if possible.

Comment (victoralmau) [reply base_comment_template/tests/test_base_comment_template.py]: Ok, i have removed everything about it.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**